dicom-archive/DICOM/DCMSUM.pm

```diff
@@ -681,7 +684,7 @@ sub confirm_single_study {
 	foreach my $studyUID (keys(%hash)) {
 	    print "'$studyUID'\n";
 	}
-	exit 33; 
+	exit $NeuroDB::ExitCodes::FILE_TYPE_CHECK_FAILURE;
```

Comment:
just nitpicking...
is this really a file_type failure? we happen to have files with 2 studyUIDs in the upload... so the file contents (headers) are wrong, but not the file type as it is DICOM...

Reply:
I'll create a new exit code for NOT_A_SINGLE_STUDY.